src/openpgp.js

```diff
@@ -311,6 +312,21 @@ export function decrypt({ message, privateKeys, passwords, sessionKeys, publicKe
     result.data = format === 'binary' ? decrypted.getLiteralData() : decrypted.getText();
     result.filename = decrypted.getFilename();
     linkStreams(result, message);
+    if (expectSigned) {
+      if (publicKeys.length === 0) {
+        throw new Error('public keys are required to verify message signatures');
```

Comment:
This is very nitpicky but please capitalize these error messages
```suggestion
        throw new Error('Public keys are required to verify message signatures');
```
(also the other ones)

Reply:
But these errors are wrapped and appear after "Error decrypting message:"

Reply:
Yeah, but the same thing is true for most other errors in this library, and we capitalize those